doc/descriptors.md

```diff
@@ -243,7 +243,18 @@ Often it is useful to communicate a description of scripts along with the
 necessary private keys. For this reason, anywhere a public key or xpub is
 supported, a private key in WIF format or xprv may be provided instead.
 This is useful when private keys are necessary for hardened derivation
-steps, or for dumping wallet descriptors including private key material.
+steps, for signing transactions, or for dumping wallet descriptors
+including private key material.
+
+For example, after importing the following 2-of-3 multisig descriptor
+into a wallet, one could use `signrawtransactionwithwallet`
+to sign a transaction with the first key:
+```
+sh(multi(2,xprv9s21Z.../84'/0'/0'/0/0,0280007cb...,033af7e98...))#lx65wk3z
```

Comment:
nit: `xprv9s21Z...` could be changed to `xprv...` and the following pubkey values could be changed to `xpub1...`, `xpub2...`, etc. since the exact values are unimportant here. The checksum can also be omitted as well

Reply:
I agree with the suggestion, is this sufficient?
`sh(multi(2,xprv.../84'/0'/0'/0/0,xpub1...,xpub2...))`

Reply:
I did make the changes suggested above